Source/Core/VideoCommon/AbstractTexture.h

```diff
@@ -31,5 +49,10 @@ class AbstractTexture
   const TextureConfig& GetConfig() const;
 
 protected:
+  virtual std::optional<RawTextureInfo> MapFullImpl();
+  virtual std::optional<RawTextureInfo> MapRegionImpl(u32 level, u32 x, u32 y, u32 width,
+                                                      u32 height);
+  bool m_currently_mapped;
```

Comment:
shouldn't this initialize to `false` by default?

Reply:
It is set in the constructor.  I still don't know what the best practice is of when to include it in the header vs constructor body.  I'm happy to add it here though.

(Looking it up, it seems the reason in-header initialization is more beneficial is if you have multiple constructors)

Reply:
>  I still don't know what the best practice is of when to include it in the header vs constructor body. I'm happy to add it here though

Rule of thumb for the codebase is if a member variable's initial value isn't dependent on any parameters of the constructor (i.e. if it's a literal or fixed value), it should just be assigned in the body. It's straightforward, and people don't have to comb through a cpp file just to see what its initial value is.